__tests__/commands/add.js

```diff
@@ -42,20 +42,20 @@ const runAdd = buildRun.bind(
   },
 );
 
-test.concurrent('add without --ignore-workspace-root-check should fail on the workspace root', async () => {
-  await runInstall({}, 'simple-worktree', async (config, reporter): Promise<void> => {
+test.concurrent('add without --ignore-workspace-root-check should fail on the workspace root', (): Promise<void> => {
```

Comment:
any reason for switching from async to promises?
Afaik async is used throughout the code and it would be better to keep it as is